android/lib/pages/profilePage.dart

```diff
@@ -0,0 +1,721 @@
+
+import 'package:android/widgets/feed_container.dart';
+import 'package:flutter/material.dart';
+
+import '../config/app_routes.dart';
+import '../widgets/form_app_bar.dart';
+import 'package:android/models/models.dart';
+import 'package:android/data/data.dart';
+
+
+class Item {
+  Item(this.name,this.icon);
+  String name;
+  Icon icon;
+
+  @override
+  bool operator ==(Object other) => other is Item && other.name == name;
+}
+
+class ProfilePage extends StatefulWidget {
+  const ProfilePage({Key? key}) : super(key: key);
+
+  @override
+  State<ProfilePage> createState() => _ProfilePageState();
+}
+
+class _ProfilePageState extends State<ProfilePage> {
+
+  final url = tom.imageUrl;
```

Comment:
"tom" instance should not be hardcoded, it should be passed as a parameter, depending on the profile page that the user is trying to view.

Reply:
done. I created a user in the homepage in order to test whether the page can get the user and the result shows that it can.